metadata-ingestion/source_docs/file_lineage.md

```diff
@@ -0,0 +1,76 @@
+# File Lineage
```

Comment:
suggest title-ing this to `File Based Lineage`

Reply:
Accepted